src/Client.js

```diff
@@ -327,7 +350,26 @@ Client.prototype._execute = function(method, path, data, query, options) {
       )
       self._handleRequestResult(response, result, options)
 
-      return responseObject['resource']
+      const metricsHeaders = [
+        'x-compute-ops',
+        'x-byte-read-ops',
+        'x-byte-write-ops',
+        'x-query-time',
+        'x-txn-retries',
+      ]
+
+      if (options && options.metrics) {
+        return {
+          value: responseObject['resource'],
+          metrics: Object.fromEntries(
+            Object.entries(response.headers).filter(([key]) =>
+              metricsHeaders.includes(key)
```

Comment:
converting the metrics to an int ourselves here we eliminate forcing that logic on each client.

Reply:
done